discovery-provider/src/queries/queries.py

```diff
@@ -1982,3 +1983,130 @@ def get_top_genre_users():
 
 
         return api_helpers.success_response({ 'user_ids': user_ids })
+
+
+
+# Get the tracks that are 'children' remixes of the requested track
+# The results are sorted by if the original artist has reposted or saved the track
+@bp.route("/remixes/<int:track_id>/children", methods=("GET",))
```

Comment:
at the very least, we should take a limit. pagination would be best

Reply:
gotcha